setup.py

```diff
@@ -7,9 +7,12 @@
 )
 
 tests_require = (
-    'nose'
+    'nose',
 )
 
+setup_requires = (
+    'flake8',
```

Comment:
Source for that change:
http://flake8.readthedocs.org/en/latest/setuptools.html

So I think it should actually be there...